lib/shopify_app/configuration.rb

```diff
@@ -29,6 +29,9 @@ class Configuration
     attr_writer :login_callback_url
     attr_accessor :embedded_redirect_url
 
+    # customize post authenticate tasks
+    attr_accessor :custom_post_authenticate_tasks
```

Comment:
Nit: would it make more sense to have a `post_authenticate_tasks` setting that defaults to the new class rather than a `custom` setting?

Reply:
yess that does make sense, but in this pre-deprecation case we _don't_ want to use the post_authenticate_tasks with a default if it's not manually set since we still want to call the old methods. I think it does make sense to have a single config with a default once we roll out the deprecation.

Reply:
Hmm yeah, fair. It wouldn't be ideal to change the setting name in the future though - so I think we should stick with what we add in this PR, whichever way we prefer to do it.